packages/test/test-end-to-end-tests/src/test/gc/gcLoadingFromOlderSummaries.spec.ts

```diff
@@ -39,6 +41,14 @@ describeNoCompat("GC loading from older summaries", (getTestObjectProvider) => {
 	let containerRuntime: IContainerRuntime;
 	let dataStoreA: ITestDataObject;
 
+	const settings = {
+		"Fluid.ContainerRuntime.Test.SummaryStateUpdateMethodV2": "refreshFromSnapshot",
```

Comment:
If you weren't going to follow up immediately and change this, I'd suggest putting a comment explaining that this is testing the old default behavior.

Reply:
The plan is to put the removal of refresh from server changes into next so that we have time, and I don't have to wait to make the changes.